src/magicgui/widgets/_concrete.py

```diff
@@ -342,7 +342,7 @@ def base(self, base: float) -> None:
 
 @backend_widget
 class ComboBox(CategoricalWidget):
-    """A dropdown menu, allowing selection between multiple choices."""
+    """A dropdown menu allowing ``Separator`` and selection between multiple choices."""
```

Comment:
```suggestion
    """A dropdown menu, allowing selection between multiple choices."""
```

i'm not sure it warrants first mention in the ComboBox description :)